### ISSUE TITLE
Create French content

### ISSUE THREAD

**Issue by jamilsinno** [jamilsinno/portfolioWebsite#29]

**Comment by CodeCrusader80** [comment]
What do you mean by create French content ? Maybe I can help at this point.

**Comment by jamilsinno** [comment]
Hey CodeCrusader, I'm looking to translate the contents of my website into French and have a button that presents the same content but in French.

**Comment by CodeCrusader80** [comment]
Hi 👋, okay it's your portfolio that's right ? I can provide you a translate, it's for the informations in your PDF file or all your work ?

**Comment by jamilsinno** [comment]
I was looking to translate the website itself, not the PDF :)

**Comment by CodeCrusader80** [comment]
It works, I'll see if I can give you a hand today with this hard work that is a French translation 😉 for the programming of the button to change the language did you do the necessary?

**Comment by jamilsinno** [comment]
Yes, or in my src folder hold the English files separately from my French ones. If you would like to fork the repo on a new branch and submit a PR, I will merge your PR once I've created the button :) Thank you for your help.

**Comment by jamilsinno** [comment]
If you would like to copy the components and add the suffix "_Fr" to the file names, that would probably be the best way to go

**Comment by CodeCrusader80** [comment]
Sorry I saw your last message too late, I hope it will be ok anyway and I wish you good luck in your project. Unfortunately Astro I do not know yet, I focus first on the rudimentary web and a little React for the moment therefore concerning the button and its programming I am not likely to be of very great help.

**Comment by jamilsinno** [comment]
No worries. Thank you so much for the translation! I will be sure to include your name and information on the README once I have some time to sit down and work on it :)

**Comment by CodeCrusader80** [comment]
You're welcome, it's my pleasure. I wish you the utmost success in your projects and your career. 😊 That's very kind of you 🤝 And if you ever happen to develop an interest in Swift, I can offer some tips and features 😉
